evaporate.js

```diff
@@ -1207,21 +1207,15 @@
                     }
                 }
 
-                var monitorPartsInterval = 0;
                 if (!bytesLoaded.length) {
```

Comment:
Not exactly a big deal but without this line isn't this polluting the global namespace?